wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -13,7 +13,9 @@ end
 
 namespace :wagn do
   desc "create a wagn database from scratch"
-  task :create do
+  task :recreate do
```

Comment:
Fwiw, you did propose *deck apps* here: https://github.com/wagn/wagn/pull/319 but that was for the same meaning as *app deck*. I strongly prefer the latter, but the good news is that even if we confuse the order, the meaning should be clear :)  Afaik, nobody wants to use one of those to mean engine deck.

I think, long term, we still want three gems, but they are not currently organized as I think they should be.

As you say, there are (and should be) two main gems: decko and cards.  But I do think decko-rails should continue to exist, only in a different form.  It's decko-rails (not decko) that should be super thin.  What I originally pushing for (and still want to see) is that decko should not depend on decko-rails.  At present we've gone the opposite route.

I get that conceptually an engine is lower level than an application.  But organizing our gems around this idea assumes that we're going to retain a railsy architecture for Decko.  That's not my assumption.  Wagn::Application may some day not inherit from Rails::Application just as Card may not inherit from ActiveRecord::Base.

Just for the hell of it, imagine that two years from now, Wagn has completely moved away from Rails.  We should still be able to plug it into rails with a rails engine.  It's an external application using our API.  Not thinking of it that way gets rails stuff baked too deeply into our architecture.

It will really help if you think of things this way.  Wagn is a pioneering new development framework supporting a MoFoS architecture.  It appreciates the Rails community, it owes a great debt to the Rails community, it hopes to contribute back to the Rails community, and it still makes use of Rails code.  But we are not committing to supporting the Rails API; we're developing the Wagn API.

What I *still* want is this:
- Card stands alone (currently failing here because of the way configuration works.  We couldn't really load paths without an external system calling Cardio#set_config with some rather involved objects from Railites)
- Decko depends on Cards and turns it into an app.  Decko does NOT depend on decko-rails, because, conceptually, why the hell would it?  This is the default case.
- decko-rails is a (thin) wrapper that makes it so that decko can plug into rails.

We're really not that terribly far away from this.  But public, spec, lib, config.... all those things should be in decko and referred to by decko-rails, not the other way around.  

I doubt you'll want to take this on, which is fine; I'm willing to.  I don't think it's going to be that hard.  But I don't think the decko-rails case should be in decko, both for functional and marketing reasons.